src/main/java/frc/robot/RobotContainer.java

```diff
@@ -60,12 +60,7 @@ private void configureDefaultCommands() {
     }
 
     private void configureButtonBindings() {
-//        rightTrigger.onTrue(new InstantCommand(() -> teleopTargetAdjustCommand = FollowPath.generatePathToAprilTag(
-//                swerveSubsystem, limelight, gyroscope
-//        )));
-//        rightTrigger.whileTrue(new ProxyCommand(() -> teleopTargetAdjustCommand));
-//        leftTrigger.onTrue(new InstantCommand(gyroscope::resetYaw));
-        a.onTrue(new InstantCommand(() -> teleopTargetAdjustCommand = FollowPath.generatePathToAprilTag(
+        rightTrigger.whileTrue(new ProxyCommand(() -> FollowPath.generatePathToAprilTag(
                 swerveSubsystem, limelight, gyroscope
         )));
```

Comment:
The decorator is shorter:
```suggestion
        rightTrigger.whileTrue(FollowPath.generatePathToAprilTag(
                swerveSubsystem, limelight, gyroscope
        ).asProxy());
```